ground-control/migrator/main.go

```
@@ -0,0 +1,109 @@
+		err := runCommand("pg_isready", "-h", dbHost, "-p", dbPort)
+		if err == nil {
+			fmt.Println("PostgreSQL is ready.")
+			return nil
+		}
+		// Sleep for a while before trying again
+		time.Sleep(10 * time.Second)
+	}
+}
+
+// Check if the database exists
+func checkDatabaseExists(dbName, dbUser, dbPassword string) (bool, error) {
```

golangci-lint (1.64.8)

func `checkDatabaseExists` is unused

(unused)